opentok/opentok.py

```diff
@@ -1599,6 +1600,164 @@ def set_broadcast_layout(self, broadcast_id, layout_type, stylesheet=None, scree
         else:
             raise RequestError("OpenTok server error.", response.status_code)
 
+    def start_render(self, session_id, opentok_token, url, max_duration=7200, resolution="1280x720", status_callback_url=None, properties: dict = None):
+        """
+        This method starts a render for the specified session.
```

Comment:
Please copy (with any required edits) the docs for the REST method: https://tokbox.com/developer/rest/#starting_experience_composer (for this description and the docs for the options). For example:

```suggestion
        This method starts a render for the specified session.
        Starts an Experience Composer for the specified OpenTok session.
        For more information, see the
       `Experience Composer developer guide <https://tokbox.com/developer/guides/experience-composer>`_.
```

Reply:
Have added this